app-frontend/src/app/pages/admin/organization/users/users.module.js

```diff
@@ -70,12 +69,25 @@ class OrganizationUsersController {
                 this.updatePagination(response);
                 this.lastUserResult = response;
                 this.users = response.results;
+                if (this.isEffectiveAdmin) {
+                    this.setUserActonButtons();
+                }
                 this.buildOptions();
             }, () => {
                 this.fetching = false;
             });
     }
 
+    setUserActonButtons() {
```

Comment:
I think this logic could be included in the `buildOptions` method so that we only iterate over the user list once.